### PR TITLE
Tweak styling of html doxygen output

### PR DESCRIPTION
**Pull request by thomasspriggs**
Tweak styling of html doxygen output

### CODE REVIEW

File: src/cbmc_style.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+code {
+	color: red;
+	background-color: WhiteSmoke;
+	border-style: solid;
+	border-width: 1px;
+	border-color: LightGrey;
+	padding-top: 1px;
```
Comment (peterjwest) [inline src/cbmc_style.css]: You can combine these lines `padding: 1px 4px;`

**Review by peterjwest — APPROVED** [review]
Seems reasonable

File: src/doxyfile [hunk 2]
```diff
@@ -1133,7 +1133,7 @@ HTML_STYLESHEET        =
 # list). For an example see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_EXTRA_STYLESHEET  =
+HTML_EXTRA_STYLESHEET  = cbmc_style.css
```
Comment (peterschrammel) [inline src/doxyfile]: Please move the .css into the doc directory.

Comment (peterschrammel) [reply src/doxyfile]: Will doxygen copy the .css into the doc/html dir?

Comment (thomasspriggs) [reply src/doxyfile]: I have moved the `.css` file into `/doc/assets`. Doxygen will copy the file into the `doc/html` directory on building, as it did before when it was in the `/src` directory.

**Comment by thomasspriggs** [comment]
@peterschrammel Re-review?

**Review by peterschrammel — APPROVED** [review]

**Pull request merged**